test/JDBC/input/views/sys-syscolumns-vu-prepare.sql

```diff
@@ -22,11 +22,12 @@ AS
 BEGIN
         DECLARE @object_name VARCHAR(50);
 
-        SET @object_name = (SELECT relname from pg_class where oid = @Oid);
+        SET @object_name = (SELECT proname from pg_proc where oid = @Oid);
```

Comment:
Had to change this in order to prevent an unrelated failing test. This test fails when I add the dependency test (sys-all-parameters-dep in schedule latest) and the error can be found here: https://github.com/babelfish-for-postgresql/babelfish_extensions/actions/runs/3139369276/jobs/5099716446. 

However, it does not seem like my changes would affected this as I did not touch syscolumns. The error from the job is that it gets the name of a view first instead of a proc created in sys-syscolumns-vu-prepare. Changing this part of the test should be okay since this is only a helper function and quries from pg_proc and pg_class (does not call syscolumns as well). The test passes when the helper function searches for pg_proc first instead of pg_class.

Reply:
To add, this error also appeared in this PR as well (https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/540). Here is an example of the failed workflow: https://github.com/babelfish-for-postgresql/babelfish_extensions/actions/runs/3147533215/jobs/5117129890 . It has the same error as well so it does not look like it's related to changes from either of these repos.